src/utils/shorten-url.js

```diff
@@ -4,7 +4,9 @@
 
 // @flow
 
-const accessToken = 'b177b00a130faf3ecda6960e8b59fde73e902422';
+const SERVER_HOST = 'https://api.profiler.firefox.com';
+const ACCEPT_HEADER_VALUE = 'application/vnd.firefox-profiler+json;version=1.0';
```

Comment:
It would be nice to share these constants with the publishing code implemented in #2360.